tox.ini

```diff
@@ -42,7 +42,9 @@ commands =
 [testenv:checkqa]
 basepython = python3.6
 commands =
-    flake8 --inline-quotes "double" pinax
+    flake8 pinax
+    isort pinax/**/*.py -c
```

Comment:
Duplicated.

I think it is better to just run `tox -e checkqa` as a single job.